app/models/repo.rb

```diff
@@ -1,9 +1,9 @@
 class Repo
   attr_reader :name,
-              :html_url
+              :url
```

Comment:
hopefully this doesnt become confusing in the future

Reply:
exactly! It did help us to clarify for ourselves the difference between :html_url and :url in the parsed JSON so that is probably maybe a good sign